src/sctools/gtf.py

```diff
@@ -0,0 +1,147 @@
+from collections import Iterable
+import string
+from . import reader
+
+
+class Record:
+    """
+    Simple namespace object that makes the fields of a GTF record available. Subclassed
+    to create records specific to exons, transcripts, and genes
+    """
+
+    __slots__ = ['_fields', '_attribute']
+
+    _del_letters = string.ascii_letters
+    _del_non_letters = ''.join(
+        set(string.printable).difference(string.ascii_letters))
+
+    def __init__(self, record):
+        """
+        :param str record: input record from file
+        """
+        fields = record.strip(';\n').split('\t')
+        self._fields = fields[:8]
+        self._attribute = {
```

Comment:
Worth a comment explaining the parsing that's happening here, and/or a short example of what fields[8] looks like.